src/configs/connectionChainInformation.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges UI.
+//
+// Parity Bridges UI is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Parity Bridges UI is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
+
+import { WsProvider } from '@polkadot/api';
+
+import { checkEnvVariable } from '../util/envVariablesValidations';
+import { createPolkadotJsUrl } from '../util/createPolkadotJsUrl';
+import { getSubstratePairConfigs } from './substrateConfigs';
+import { ConnectionChainInformation } from '../types/sourceTargetTypes';
+
+const getProvider = (provider: string) => new WsProvider(provider);
+
+const getConnectionChainInformation = (chainNumber: string): ConnectionChainInformation => {
```

Comment:
as mentioned in previous comment - I'm trying to understand the use of this file

Reply:
explained above.

Reply:
The use of this file is to generate dynamically and agnostically  all the necessary values for constructing the API connection so then we can read the on-chain metadata config values. I say agnostically because at this point there is not `Rialto` or `Millau` , there are `chain1` and `chain2` that are filled with the values that comes from the `getSubstratePairConfigs ` that this yes generates specific set up for Rialto and Millau. 

I know that for only one pair all this complexity does not have too much sense , but if we add a new pair in the future, let's say MIllau<>Westend, then is just a matter of adding a new case in the file `src/configs/substrateConfigs/index.ts`. As i mentioned before, in this file we should provide specific chain related information.

Anyway, i think is better to move the `provider` and `polkadotjsUrl`to the file `src/configs/substrateConfigs/index.ts` so this file does not handle any construction logic.

Reply:
After reviewing the code , i think this file was redundant since i was doing the similar logic in two files. So i removed this one and i unified everything on file: `src/configs/index.ts`